src/installer-source/KolibriSetupScript.iss

```diff
@@ -585,22 +607,26 @@ begin
             'System\CurrentControlSet\Control\Session Manager\Environment',
             'KOLIBRI_SCRIPT_DIR'
         )
-
+        PipDir := GetPipDir('')
         Exec('cmd.exe', '/c "reg delete HKCU\Environment /F /V KOLIBRI_SCRIPT_DIR"', '', SW_HIDE, ewWaitUntilTerminated, ErrorCode)
         { Must set this environment variable so the systray executable knows where to find the installed kolibri.exe script}
         { Should by in the same directory as pip.exe, e.g. 'C:\Python33\Scripts' }
         RegWriteStringValue(
             HKLM,
             'System\CurrentControlSet\Control\Session Manager\Environment',
             'KOLIBRI_SCRIPT_DIR',
-            GetPipDir('')
+            PipDir
         );
         RegDeleteValue(
             HKLM,
             'System\CurrentControlSet\Control\Session Manager\Environment',
             'KOLIBRI_SETUP'
         )
-        Exec('cmd.exe', '/c "reg delete HKCU\Environment /F /V KOLIBRI_SETUP"', '', SW_HIDE, ewWaitUntilTerminated, ErrorCode)   
+        Exec('cmd.exe', '/c "reg delete HKCU\Environment /F /V KOLIBRI_SETUP"', '', SW_HIDE, ewWaitUntilTerminated, ErrorCode)
+        {Set kolibri command to Path}
+        
+        EnvAddPath(PipDir) 

```

Comment:
Agreeing with @intelliant01 's input from #161  :

> Since KOLIBRI_SCRIPT_DIR system variable is being set, it should be used as %KOLIBRI_SCRIPT_DIR% to prepend to the Path variable (User or System variables) to ensure a single source of truth. May also be helpful in all cases wherein the Path variable needs to be checked. Once case that I can think of is clean-up during uninstallation.

Reply:
...noting that I have no idea when the variable substitution occurs. If it's expanded immediately as the value is inserted, it's useless. But if variable substitution happens each time the registry entry is read, it makes sense for having a "single source of truth".